java-gradle/java-dynamic-loading/Jenkinsfile

```diff
@@ -0,0 +1,8 @@
+regressionTest {
+    FILE_PATH =  "src/main/java/com/rookout/HelloWorld.java"
+    LINE_NUM =  "16"
+    SHA =  "7459814efb8eb2c0caf3956e441f159252ece842aa714be32ad350dd5e331d9b"
+    URL_FOR_TRIGGER = "http://localhost:7000/hello/ori"
+    LANG = "Java"
+    BACKGROUND_IMAGE_CMD = "unset JAVA_TOOL_OPTIONS && java -jar rookoutDemo-1.0.0.jar & export ROOKOUT_TARGET_PID=$(ps -ef | grep 'java -jar rookoutDemo-1.0.0.jar' | grep -v grep | awk '{print $2}') && java -jar rook.jar &"
```

Comment:
@MickaelAlliel the rook.jar is in the same folder?

Reply:
and JAVA_TOOL_OPTIONS is set ..... right?

Reply:
rook.jar is located in 
`$JENKINS_AGENT_WORKDIR/rook.jar` which technically should be the root dir of that yes